src/main/resources/interface-specification.yml

```diff
@@ -737,85 +700,50 @@ components:
       schema:
         type: string
   schemas:
-    KeysResponse:
-      type: object
-      properties:
-        keys:
-          type: array
-          items:
-            $ref: '#/components/schemas/ClientKey'
-      required:
-        - keys
-    Keys:
-      type: array
-      items:
-        $ref: '#/components/schemas/KeySeed'
-    KeySeed:
```

Comment:
`KeySeed` must be kept because it is used for key creation. The seed is different from the returned value.
For example the seed cannot contain the `kid` because it is generated by this service on the creation.
(the aggregated model may be called `KeysSeed` instead of `KeysResponse`)